tests/lax_numpy_operators_test.py

```diff
@@ -654,6 +655,11 @@ def testShiftOpAgainstNumpy(self, op, dtypes, shapes):
     shift_rng = jtu.rand_int(self.rng(), high=max(info.bits, shift_info.bits))
     args_maker = lambda: (x_rng(shapes[0], dtype), shift_rng(shapes[1], shift_dtype))
 
+    np_version = version.parse(np.__version__)
+    
+    if np_version.major < 2 and op.__name__ in ("bitwise_left_shift", "bitwise_right_shift"):
```

Comment:
Two suggestions:
- we have a utility for this already: you can check that `jtu.numpy_version() < (2, 0, 0)`
- rather than skipping the test, you could use `left_shift`/`right_shift`, which should be equivalent functions defined in older NumPy versions. That way we don't lose test coverage for the new methods.